.github/ISSUE_TEMPLATE/package_build_request.yml

```diff
@@ -0,0 +1,105 @@
+name: Package build request
+description: Request a package to be built (e.g. new, updates, and rebuilds).
+labels: ['package_request']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please use this form to submit a request for a package to be built.
+
+        We kindly ask that you take the time to:
+          - Search Anaconda.org ([anaconda](https://anaconda.org/anaconda) channel) to verify whether the package already exists in our channels.
+          - Search through [existing issues](https://github.com/ContinuumIO/anaconda-issues/issues) to verify whether someone else has already requested the same package.
+          - Fill in all of the fields below, as best you can.
+          - Review and comply with our [Anaconda Community Code of Conduct](https://www.anaconda.com/community-code-of-conduct/).
+
+        Thank you for your contribution and for helping Anaconda better serve its community of users. 🐍
+
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Checklist
+      description: Please confirm and check all the following options.
+      options:
+        - label: I added a descriptive title
+          required: true
+        - label: I searched for other package requests and couldn't find a duplication
+          required: true
+
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Checklist
+      description: "Please confirm the following:"
+      options:
+        - label: I added a descriptive title (including the requested package **name**, **version**, and **platform**).
+          required: true
+        - label: I searched the [anaconda](https://anaconda.org/anaconda) channel and did not find the package I'm requesting.
+          required: true
```

Comment:
I don't think we want to point them at anaconda.org/anaconda - it would be better to point them at https://repo.anaconda.com/pkgs/main/ and https://repo.anaconda.com/pkgs/r/

Reply:
Done. Thank you!